api/openapi/v2/core-command.yaml

```diff
@@ -347,8 +367,16 @@ paths:
                 items:
                   $ref: '#/components/schemas/IssueCommandResponse'
   /config:
-    get:
+    post:
       summary: "Returns the current configuration of the service."
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
```

Comment:
I don't think we need an array of requests here. Probably will only service one request at a time and no array of responses is currently returned.

Comment applies to /config, /ping, /version and /metrics endpoints on all services.

Reply:
Any use-case endpoint takes either a single use-case request or an array of use-case requests (all of the same type).  I believe both the request and response sections of the specification need to be modified to reflect this (`anyOf` per conversation [here](https://github.com/edgexfoundry/edgex-go/pull/2324#discussion_r371374159)).

Reply:
We discussed offline rationale for changing endpoints from GET to POST (these endpoints do not uniquely identify a specific resource and we don't want their output to be cacheable). In changing the endpoints to POST, the above was done for consistency's sake.